## Welcome to PacBio's GitHub space ðŸ‘‹

Are you looking for great tools to analyze your HiFi data?
 - [Tandem repeat genotyping and visualization from PacBio HiFi data](https://github.com/PacificBiosciences/trgt)
 - [A battery of methylation tools for PacBio HiFi reads](MethBat)
 - [Small variant, structural variant, and short tandem repeat phasing tool](https://github.com/PacificBiosciences/HiPhase)
 - [Fusion gene caller for Iso-Seq sequencing data.](https://github.com/PacificBiosciences/pbfusion)
 - [HLA star-calling tool for PacBio HiFi data types](https://github.com/PacificBiosciences/hifihla)
 - [Iso-Seq - Scalable De Novo Isoform Discovery](https://github.com/PacificBiosciences/IsoSeq)
 - [A minimap2 frontend for PacBio native data formats](https://github.com/PacificBiosciences/pbmm2)

How about complete workflows for your HiFi data?
 - [HiFi target enrichment workflow](https://github.com/PacificBiosciences/HiFiTargetEnrichment)

Help pages for tools that are running on our long-read platoform?
 - [Kinnex HiFi read splitter](https://skera.how/)
 - [CCS: Generate Highly Accurate Single-Molecule Consensus Reads (HiFi Reads)](https://ccs.how/)
 - [Predict 5mC in PacBio HiFi reads](https://github.com/PacificBiosciences/jasmine)
